src/functions-and-arrays.js

```diff
@@ -1,38 +1,109 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1>num2){
+    return num1;
+  } else {
+    return num2;
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+  if (words.length===0){
+    return null;
+  }
+ let longestWord = words[0]
+
+  for (let i = 0; i < words.length; i++) {
```

Comment:
```suggestion
  for (let i = 1; i < words.length; i++) {
```
Fijate, al principio le estas metiendo la posicion 0, es decir que vas a compararlo consigo misma, pues para ahorrarnos esa vuelta le ponemos la siguiente posicion y que tire millas :D